pkgs/applications/graphics/ovito/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitLab, cmake
+, boost, netcdf, hdf5, fftwSinglePrec, muparser, openssl, ffmpeg, python
+, qtbase, qtsvg, qttools, qscintilla }:
+
+stdenv.mkDerivation rec {
+  pname = "ovito";
+  version = "3.4.0";
+
+  src = fetchFromGitLab {
+    owner = "stuko";
+    repo = "ovito";
+    rev = "v${version}";
+    sha256 = "1y3wr6yzpsl0qm7cicp2mppfszxd0fgx8hm99in9wff9qd0r16b5";
+  };
+
+  buildInputs = [
+    cmake
```

Comment:
Sorry that I missed it but cmake belongs into nativeBuildInputs.

Reply:
Not a problem. Thanks for reviewing things. I've fixed that now.

Gave me a chance to fix having used `stdenv.mkDerivation` instead of `mkDerivaiton` too (later is overridden to properly wrap Qt applications).